playbook/app/pb_kits/playbook/pb_icon/docs/_icon_custom.md

```diff
@@ -1,19 +1,14 @@
 # Tips for Custom Icons
 
-When using custom icons it is important to introduce a "clean" SVG. In order to ensure these custom icons perform as intended within your kit(s), ensure these things have been modified from the original SVG markup:
-
-Attributes must be React compatible e.g. <code>xmlns:xlink</code> should be <code>xmlnsXlink</code> and so on. <strong>There should be no hyphenated attributes and no semi-colons!.</strong>
-
-Fill colors with regards to <code>g</code> or <code>path</code> nodes, e.g. <code>fill="black"</code>, should be replaced with   <code>currentColor</code> ala <code>fill="currentColor"</code>. Your mileage may vary depending on the complexity of your SVG.
-
-Pay attention to your custom icon's dimensions and `viewBox` attribute. It is best to use a `viewBox="0 0 512 512"` starting point __when designing instead of trying to retrofit the viewbox afterwards__!
-
-You must source *your own SVG into component/view* you are working on. This can easily be done in programmatic and maintainable ways.
-
```

Comment:
Isn't some of this still needed? 

`fill="black"`, should be replaced with `currentColor` ala `fill="currentColor"`

I believe we still need something like that to make colors respond to text size.

Reply:
this has been returned